en/docs/concepts/authorization/grant-types.md

```diff
@@ -1,13 +1,27 @@
 # Grant Types
 
-In OAuth2.0 the term `grant type` refers to the way a client gets an access token.
+In OAuth2.0 the term **grant type** refers to the way a client gets an access token.
 The [specification](https://tools.ietf.org/html/rfc6749) defines four main grant types.
 
-[1. Authorization code grant type](authorization-code.md)
+[- Authorization code grant type](authorization-code.md)
 
-[2. Implicit grant type](insertlink)
+[- Implicit grant type](implicit.md)
 
-[3. Resource owner grant type](insertlink)
+[- Resource owner grant type](resource-owner.md)
 
-[4. Client credential grant type](insertlink)
+[- Client credential grant type](client-credential-grant.md)
+
+
+Apart of the above four grant types, WSO2 Identity Server supports for few extension grants as;
+
+[- JWT grant type](insertlink)
+
+[- SAML bearer grant type](insertlink)
+
+[- Kerberos grant type](insertlink)
+
+
+To renew the existing access token, WSO2 Identity Server supports for;
```

Comment:
To renew the existing access token, WSO2 Identity Server supports the [refresh token grant type](refresh-token-grant.md)